mailjet_simple.module

```diff
@@ -328,3 +334,158 @@ function mailjet_simple_unsubscribe_all_lists($account) {
     }
   }
 }
+
+/**
+ * Helper function to create a new campaign for a Mailjet list.
+ *
+ * @param integer $list_id
+ *    List ID to associate with the campaign.
+ *
+ * @param array $body
+ *    Message to used for campaign creation.
+ *
+ * @return array $result
+ *    Returns Mailjet API call result.
+ */
+function mailjet_simple_create_campaign($list_id, $body) {
```

Comment:
@andyg5000 , should I rename this to mailjet_simple_create_campain_draft() since we are updating the public function from createCampaign to createCampaignDraft?